conformance-pack.tf

```diff
@@ -0,0 +1,30 @@
+  ## The StratusGrid recommended AWS managed Conformance Pack can be added here as part of Starter script. 
+locals {
+  stratusgrid_recommended_aws_provided_defaults = {
```

Comment:
Will this module be released publicly? If so, I would remove all SG branding from the locals and elsewhere

Reply:
Looks like couple modules are public. Who makes that decision? @rchildress87